src/lib/create/from-string.js

```diff
@@ -215,8 +214,12 @@ export function configFromString(config) {
         return;
     }
 
-    // Final attempt, use Input Fallback
-    hooks.createFromInputFallback(config);
+    if (config._strict) {
+        config._isValid = false;
+    } else {
+        // Final attempt, use Input Fallback
+        hooks.createFromInputFallback(config);
+    }
```

Comment:
Why do you enforce _strict for the InputFallback?
Will this cause breaking changes for other users. Who might not be using strict for their moment usage.

Reply:
If you have strict, then you'd have a format, and then you wouldn't do the fallback. So it is not breaking for existing users.